katello.gemspec

```diff
@@ -52,6 +52,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency "pulp_container_client", ">= 1.3.0", "< 1.4.0"
   gem.add_dependency "pulp_rpm_client", ">=3.3.0", "< 3.4.0"
   gem.add_dependency "pulp_2to3_migration_client", "> 0.1", "< 0.3.0"
+  gem.add_dependency "pulp_certguard_client", "0.1.0rc5.dev01588849857"
```

Comment:
can you relax this a bit   '>= 0.1.0rc5',  '< 2.0' 

I think that will cover the upcoming 1.0 ga release of the plugin

Reply:
Oh..missed this comment..pushing now with the change..